app/src/main/java/com/keylesspalace/tusky/components/timeline/TimelineFragment.kt

```diff
@@ -217,7 +217,7 @@ class TimelineFragment :
             }
         })
 
-        lifecycleScope.launch {
+        viewLifecycleOwner.lifecycleScope.launch {
```

Comment:
There seems to be an edge case where the lifecycleScope is still valid but the Fragment view is already destroyed, this should fix it